pkg/server/server.go

```diff
@@ -251,6 +267,11 @@ func New(
 		// these are set in the kine endpoint config below
 		compactInterval = tuning.KineCompactInterval
 		pollInterval = tuning.KinePollInterval
+	} else {
```

Comment:
Can we re-organize the code in such a way that we don't need to write `append` twice? Something like:

```go
snapshotParameters := dqlite.SnapshotParams{ // declare default
	Threshold: default_threshold,
	Trailing:  default_trailing,
}
if exists, err := fileExists(dir, "tuning.yaml"); err != nil { // logic to load user decisions
... all the logic above here ....
}
options = append(options, app.WithSnapshotParams(dqlite.SnapshotParams{
	Threshold: v.Threshold,
	Trailing:  v.Trailing,
})
...
```